airflow/pipe_naf_reader_dag.py

```diff
@@ -107,7 +114,24 @@ def build(self, dag_id):
                 dag = dag
             )
 
-            dag >> source_exists >> naf_reader
+            generate_partitioned_table = KubernetesPodOperator(
+                namespace = os.getenv('K8_NAMESPACE'),
+                image = '{docker_image}'.format(**config),
+                cmds = ['./scripts/run.sh',
+                    'generate_partitioned_table_daily',
+                    '{name}'.format(**self.country),
+                    '{bq_output}'.format(**self.country),
+                    '{bq_partitioned_output}'.format(**self.country),
+                    '{ds}'.format(**config)
+                ],
+                name = 'generate-partitioned-table-{}'.format(name),
+                task_id = "generate-partitioned-table-task",
+                get_logs = True,
+                in_cluster = True if os.getenv('KUBERNETES_SERVICE_HOST') else False,
+                dag = dag
```

Comment:
@smpiano I would add a pool of `k8operators` here. A pool is define in airflow -> admin -> pools
We will be triggering multiple K8 operators, maybe setting it to 6 is good to start with.